engines/onboarding/app/controllers/onboarding/api/v1/invitations_controller.rb

```diff
@@ -2,61 +2,106 @@ module Onboarding
   module Api
     module V1
       class InvitationsController < ApplicationController
+        before_action :authenticate, only: [:bulk]
+        before_action :load_group, only: [:bulk]
+        before_action :load_invitation, only: [:show, :accept]
 
-        before_action :authenticate
-        before_action :load_group, only: [:create]
-        before_action :load_invited_user, only: [:create]
+        # Creates invitations in bulk
+        #
+        # POST /api/v1/invitations/bulk
+        #
+        def bulk
+          authorize @group
+
+          emails = bulk_params[:emails] || ''
+
+          if emails.empty?
+            return render(
+              status: :bad_request,
+              json: {errors: {emails: t('onboarding.invitation.bulk.errors.empty') }}
+            )
+          end
+
+          emails = emails.split(',')
+          valids = valid_emails(emails)
```

Comment:
line 16 is when `emails` fields is `null`. That can happen if user submit the form empty